rest_framework/schemas/openapi.py

```diff
@@ -564,3 +582,27 @@ def _get_responses(self, path, method):
                 'description': ""
             }
         }
+
+    def _get_tags(self, path, method):
+        # If user have specified tags, use them.
+        if self._tags:
+            return self._tags
+
+        # Extract tag from viewset name
+        # UserView         tags = [User]
+        # User             tags = [User]
+        if hasattr(self.view, 'action'):
+            name = self.view.__class__.__name__
+            if name.endswith('ViewSet'):
+                name = name[:-7]
+            elif name.endswith('View'):
+                name = name[:-4]
+            return [name]
```

Comment:
Paths are most likely going to be all lowercase if they are used as a fallback. For consistency, do we want to lowercase the view-generated name here?

Reply:
Sure. Thanks for the suggestion. :)